### PR TITLE
Feature/jude/user logout

### PR DESCRIPTION
**Pull request by judeokafor**
Create logout route
Fix return error issue
Merge from branch master

### CODE REVIEW

File: functions/src/controllers/locations.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ const Locations = {
         }
     },
 
+    // eslint-disable-next-line max-lines-per-function
```
Comment (codemutatio) [inline functions/src/controllers/locations.js:53]: Remove this eslint disable comment

File: functions/src/controllers/users.js [hunk 2]
```diff
@@ -108,12 +109,28 @@ class userController {
...
+  * @function
+  * @param {object} req - request object
+  * @param {object} res - response object
+  * @return  {Object} result
+  */
+    static async logoutUser(req, res) {
+        try {
+            const { userId } = req.user;
+            await firebase.auth().signOut();
+            return successNoData(res, 200, 'Logout successfully');
```
Comment (enyeInc) [inline functions/src/controllers/users.js:131]: use http-status-code for 200

File: functions/src/utils/errorHandler.js [hunk 3]
```diff
@@ -1,3 +1,6 @@
+const {
+    CREATED, getStatusText, INTERNAL_SERVER_ERROR, NOT_FOUND, OK,
+} = require('http-status-codes');
 class ErrorHandler {
     static validationError(res, message) {
         return res.status(400).json({ message, status: 'error' });
```
Comment (enyeInc) [inline functions/src/utils/errorHandler.js:6]: use http-status-code for 400

**Review by enyeInc — APPROVED** [review]
minor fixes but LGTM

**Pull request merged**